content/blog/2021/09/running-rego-policies.md

```diff
@@ -0,0 +1,141 @@
+but the differences end there.
+
+By targeting Web Assembly, with the `opa build` command, it's possible
+to build policies targeting Open Policy Agent and Gatekeeper in a
+format that is understood and runnable by Kubewarden: `wasm`.
+
+Having presented the context, we can now go for the good news:
+Kubewarden supports both **Open Policy Agent** and **Gatekeeper**
+policies! -- **without any change needed on your existing policies**.
+
+#### A burrito, with some Rego inside of it
+
+[Flavio](https://twitter.com/flavio_castelli) had the idea and did the
+initial work of implementing a runtime for Rego policies built for the
+Web Assembly target inside Kubewarden. As you can imagine, the only
+logical name one can think of for a Burrito full of Rego can be
+`burrego`. This project is currently embedded into our
+`policy-evaluator` project in a crate of its own. If there is
+community interest, we can publish it as an individual crate on
+`crates.io`, so let us know if this is of your interest.
+
+## Distributing policies
+
+### Building policies
+
+Despite your policy code does not need to be adapted at all, the first
+task that needs to be done is to use `opa build` to build each one of
+your policies to a single Web Assembly file.
+
+`opa build` is able to accept any number of source files, if you
+happen to have your policy composed by different files. Each file that
+is provided as an argument to the `opa build` command can be imported
+normally through the `data` document inside of your policy.
+
+### Annotating policies
+
+After we have a Web Assembly target file, Kubewarden does not know
+what execution model it should use when instantiated.
+
+We have some tricks to identify whether a policy was built with one of
+our SDK's, but we don't have those tricks to tell the difference
+between a Rego policy targeting Open Policy Agent and another one
+targeting Gatekeeper.
+
+We could have implemented some heuristics, but we thought it was best
+to ask our users to annotate their policies, as they are doing
+today. After all, all policies that run in the `policy-server` need to
+be annotated first, and policies have to provide extra metadata. More
+on that right now.
+
+The annotation process is a very simple way of extending the Web
+Assembly binary with custom headers that tells Kubewarden a little
+more about some things: like what are the resources this policy
+targets, the version of the policy, author, license... some other
+things, and now: the execution model.
+
+After you have built your Rego policy with `opa build`, regardless of
+whether it's targeting Open Policy Agent or Gatekeeper, you must
+annotate the Web Assembly binary to specify what execution model it has.
+
+### Pushing policies
+
+If you followed the steps until now, now Kubewarden has a Web Assembly
+binary with some metadata --that travels with the binary itself,
+nice!--.
+
+Now, you can push your policy normally with `kwctl push` to an OCI
```

Comment:
Nit:

```suggestion
Now, you can push your policy as usual with `kwctl push` to an OCI
```

I think "normally" as an adverb can't be in that position, would need to be "you can push normally your policy as with kwctl" or similar.